libs/design/switch/src/switch/switch.component.html

```diff
@@ -1,12 +1,23 @@
-<label class="daff-switch">
+<label class="daff-switch" 
+  [class.daff-switch--disabled]="disabled"
+  [class.daff-switch--position-left]="position === 'left'"
```

Comment:
I'm slowly making updates to changes in class names so they're not so long. These can be updated to `class.disabled`, `class.left`, etc.